### PR TITLE
Add WELTRAJ test case

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**
Depends on https://github.com/OPM/opm-common/pull/3465

### CODE REVIEW

File: spe1/SPE1CASE1_WELTRAJ.DATA [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+-- The INIT keyword is used to request an .INIT file. The .INIT file
+-- is written before the simulation actually starts, and contains grid
+-- properties and saturation tables as inferred from the input
+-- deck. There are no other keywords which can be used to configure
+-- exactly what is written to the .INIT file.
+INIT
+
+
+-- -------------------------------------------------------------------------
+NOECHO
```
Comment (tskille) [inline spe1/SPE1CASE1_WELTRAJ.DATA:78]: Keyword NOECHO is not supported. I'm in favor for removing this from the deck.

File: spe1/SPE1CASE1_WELTRAJ.DATA [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+
+PERMY
+-- Equal to PERMX
+	100*500 100*50 100*200 /
+
+PERMZ
+-- Cannot find perm. in z-direction in Odeh's paper
+-- For the time being, we will assume PERMZ equal to PERMX and PERMY:
+	100*500 100*50 100*200 /
+ECHO
```
Comment (tskille) [inline spe1/SPE1CASE1_WELTRAJ.DATA:110]: This can hopefully also be removed as we are not supporting this.

**Review by tskille — CHANGES_REQUESTED** [review]
Two minor comments regarding ECHO and NOECHO. 

I would also suggest that we move this test to a new folder weltraj and perhaps rename the test to WELTRAJ-01.DATA

**Comment by tskille** [comment]
I see now that this PR includes changes to [co2store/CO2STORE_GASWAT.DATA](https://github.com/OPM/opm-tests/pull/940/commits/496f5aeb0ce136427e42b76463ba13c307042fcc#diff-23c8d86200860645c28eae78514c79ad03b72b65a9fa12a62fb56e846ffc4ea9) . Is this intentional ? 

Apart from this I'm happy to merge this now.

**Comment by goncalvesmachadoc** [comment]
Thank you for noticing. Problem solved.

**Review by tskille — APPROVED** [review]
Excellent, merging now

**Pull request merged**